Formula/mikutter.rb

```diff
@@ -0,0 +1,378 @@
+          ruby_bin = homebrew_ruby if File.exist?(homebrew_ruby)
+
+          exec(mikutter_env, ruby_bin, "#{libexec}/mikutter.rb", *ARGV)
+        end
+    EOS
+  end
+
+  def info_plist
+    <<-EOS.undent
+      <?xml version="1.0" encoding="UTF-8"?>
+      <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+      <plist version="1.0">
+      <dict>
+        <key>CFBundleDevelopmentRegion</key>
+        <string>ja_JP</string>
+        <key>CFBundleDisplayName</key>
+        <string>mikutter</string>
+        <key>CFBundleExecutable</key>
+        <string>mikutter</string>
+        <key>CFBundleIconFile</key>
+        <string>mikutter.icns</string>
+        <key>CFBundleIdentifier</key>
+        <string>net.hachune.mikutter</string>
+        <key>CFBundleInfoDictionaryVersion</key>
+        <string>6.0</string>
+        <key>CFBundleName</key>
+        <string>mikutter</string>
+        <key>CFBundlePackageType</key>
+        <string>APPL</string>
+        <key>CFBundleShortVersionString</key>
+        <string>#{version}</string>
+        <key>CFBundleVersion</key>
+        <string>#{version}</string>
+        <key>LSApplicationCategoryType</key>
+        <string>public.app-category.social-networking</string>
+        <key>NSHumanReadableCopyright</key>
+        <string>Copyright © #{Time.now.strftime("%Y")} toshi_a. All rights reserved.</string>
+      </dict>
+      </plist>
+    EOS
+  end
+
+  def compatibility_plugin
+    <<-EOS.undent
+      # -*- coding: utf-8 -*-
+
+      Plugin.create(:macos_compatibility) do
+        require "shellwords"
+
+        ###########
+        #  Sound  #
+        ###########
+        defsound(:afplay, "afplay") do |filename|
+          if FileTest.exist?(filename) && command_exist?("/usr/bin/afplay")
+            bg_system("/usr/bin/afplay", filename)
+          end
+        end
+        # set as default
+        UserConfig[:sound_server] ||= :afplay
+
+        ###################
+        #  Notifications  #
+        ###################
+        on_popup_notify do |user, text, &stop|
+          SerialThread.new do
+            begin
+              command = notify_command(user, text)
+              bg_system(*command)
+            rescue => e
+              error e
+            end
+          end
+          stop.call
+        end
+
+        def notify_command(user, text)
+          metadata = {
+            title: "mikutter",
+            sender: "net.hachune.mikutter"
+          }
+          if user
+            user_img = Gdk::WebImageLoader.local_path(user[:profile_image_url])
+            metadata[:title] = "@\#{user[:idname]} (\#{user[:name]})"
+            metadata[:contentImage] = user_img
+          end
+          text = text.to_s if text.is_a? Message
+          metadata[:message] = text
+
+          command = [
+            "#{HOMEBREW_PREFIX}/opt/terminal-notifier/bin/terminal-notifier",
```

Comment:
You can’t rely on this being present because it’s only `:recommended`; not required.